packages/client/README.md

```diff
@@ -88,6 +88,33 @@ Setting a new admin fee:
 // Any number between 0 and 1, inclusive
 const receipt = await dataUnion.setAdminFee(0.4)
 ```
+
+If the Data Union is set up to use the [default join server](../default-join-server/README.md) then members can join the Data Union by giving a correct secret.
+
+Admin can add secrets that allow anyone to join, as well as revoke those secrets, using the following functions:
+```js
+await dataUnion.createSecret() // returns the newly created secret
```

Comment:
Add return types to these examples and the ones below:
like: `const s: string = await dataUnion.createSecret()`

Reply:
the return type is described in the next paragraph... I guess I could format it as TypeScript (or add TS examples) as well, but I think I'd prefer to make a separate ticket about that